scripts/Makefile.am

```diff
@@ -30,12 +30,10 @@ substitute_vars	= $(AM_V_GEN)sed \
 	@$(substitute_vars) < $< > $@.tmp && mv $@.tmp $@
 
 install-data-hook:
-	$(MKDIR_P) $(DESTDIR)$(sysconfig_network_scriptsdir)
```

Comment:
Where do we need `sysconfig_network_scriptsdir` then, Maybe we can also cleanup `configure.ac` and the definition above in this file?

Reply:
The move does not invalidate it completely, there may be still user scripts/hooks inside or worse,
user scripts may source function.netconfig from this location or we want to mention it in docs.
I've moved the link creation to the spec file to make it easier to conditionally not install the link
if we decide to break compatibility to existing customer scripts (a no-go for SLES, but ...).